### PR TITLE
ref #276: Add delete confirmation dialog

### PR DESCRIPTION
**Pull request by UlrichKu**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#276
| License       | MIT

### CODE REVIEW

**Comment by pixeljunkie** [comment]
I would add the missing blocking screen using the "PleaseWait()" method which is listed in #276 because it is no big deal to add and the other issue which moved to follow-up issue will be more complicated and maybe not part of the next release.

**Comment by UlrichKu** [comment]
PleaseWait() added.

File: src/MediaManagerBundle/Resources/public/js/mediaManager.js [hunk 1]
```diff
@@ -289,10 +295,12 @@
                     'id': encodeURIComponent(id)
                 },
                 error: function (responseData) {
+                    $.unblockUI();
```
Comment (pixeljunkie) [inline src/MediaManagerBundle/Resources/public/js/mediaManager.js]: Just remembered $.unblockUI() is deprecated in 6.3. As of 6.3 we need to call: CHAMELEON.CORE.showProcessingModal() and CHAMELEON.CORE.hideProcessingModal(); So either we need seperate branches, or after the upmerge a change in 6.3/7.0.

Comment (UlrichKu) [reply src/MediaManagerBundle/Resources/public/js/mediaManager.js]: Added #408 for this.

**Pull request merged**